examples/alpine/build.yaml

```diff
@@ -1,28 +1,59 @@
+#Refer README.md for more info
 packages:
-- name: "alpine"
-  release: "ThinXR_7.3.15"
-  version: "0.1.0"
+- name: "alpine" #Can be changed
+  release: "ThinXR_7.3.15" #Do not change
+  target-release: "ThinXR_7.3.15" #Can be changed
```

Comment:
What purpose does target-release serve?

Reply:
target-release, as the comment says, "can be modified" and corresponds to the release for which rpm is being built. Also this release is appended to the rpm name.